getting-started/data-setup.rst

```diff
@@ -26,7 +26,7 @@ Install the LSST Science Pipelines
 
 If you haven't already, you'll need to install the LSST Science Pipelines.
 We recommend that you install the pre-built binary packages by following the instructions at :doc:`/install/newinstall`.
-This tutorial is intended to work with the |eups-tag-mono| tag of the ``lsst_distrib`` EUPS package.
+This tutorial was developed using the ``w_2021_33`` tag of the ``lsst_distrib`` EUPS package.
```

Comment:
Maybe we can say that we expect it to work with newer releases (until such a time as it doesn't).

Reply:
My point being that we don't want to force people to downgrade to an older release just to run the tutorial if they don't need to.

Reply:
OK.  How about the new words

Reply:
Looks good. Thanks.